integration/src/test/java/org/mule/test/components/RedeliveryPolicyTestCase.java

```diff
@@ -69,19 +67,6 @@ public void hashWorksOverDataWeaveObject() throws Exception {
     assertThat(queueHandler.read("redeliveredMessageQueue", RECEIVE_TIMEOUT), notNullValue());
   }
 
-  @Test
-  public void redeliveryPolicyDoesntUseCpuLite() throws Exception {
```

Comment:
we should not lose coverage for this scenario. Parameterize it to continue executing with the proactor PS